models.yml

```diff
@@ -3705,24 +3714,12 @@ mediafile:
   create_timestamp:
     type: timestamp
     restriction_mode: A
-  is_public:
-    type: boolean
-    description: "Calculated field. inherited_access_group_ids == [] can have two causes: cancelling access groups (=> is_public := false) or no access groups at all (=> is_public := true)"
-    read_only: true
-    required: true
-    restriction_mode: A
   token:
     type: string
     restriction_mode: A
-  inherited_access_group_ids:
```

Comment:
That means on migration you have to migrate all the data to newly created meeting_mediafile instances

Reply:
Considering I have already done that in the backend PR, what point are you trying to make with this comment?

Reply:
Just to not forget! This was the first I checked in reviewing the backend side, but then I forget to delete this here.